src/main/java/co/inventorsoft/StreamHomework.java

```diff
@@ -6,62 +6,37 @@
 import java.util.List;
 import java.util.Map;
 
-/**
- * Contains simple cases for trying Stream API in action.
- */
+
 public class StreamHomework {
-    /**
-     * Used to filter children and adults.
-     * Returns collection of teenagers (13-19 years old).
-     *
-     * @param people collection of people to extract teenagers
-     * @return collection of teenagers
-     */
     public List<Person> extractTeenagers(final List<Person> people) {
-        return null;
+        return people.stream()
+                .filter(Person -> Person.getAge() >= 13 && Person.getAge() <= 19)
```

Comment:
According to java code conventions we don't use Capitalized words for naming a variable and lambda parameter in this case is exactly a variable. Please, rename it to `person` so it doesn't look like a class name.

Reply:
rename the parametr